service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -80,4 +80,6 @@
     @Valid
     private final List<@NotNull(message = "You need to add details to children")Element<Child>> children1;
     private final String childrenMigrated;
+    private final List<Element<HearingBookingDetail>> hearingDetails;
```

Comment:
I'd like to see this be `private final List <Element<HearingBooking>> hearingBookingDetails`

Reply:
👍

Reply:
@rebeccabrennan what do you think?

Reply:
we agreed to keep it hearingDetails